### DISCUSSION TITLE
Feature Request: Copying code while reviewing changes

### DISCUSSION THREAD

**Discussion by saagarjha** [community/community#51] [Mobile] [answered]
I often copy code from what's being reviewed, often to stick it in a comment or suggestion. While reviewing, however, the long press action seems to be "add comment" and I can't see a way to copy any text. It'd be nice if it was possible to do this.

**Comment by Codeize** [comment]
I strongly agree with this, it should be easy to copy code.

**Comment by candyho** [comment, marked as answer]
Hey thanks for the feedback! This is something on our radar, will keep you posted as we get updates!

**Reply by Codeize** [reply]
Any updates? @candyho
